app/assets/stylesheets/barchart.scss

```
@@ -0,0 +1,43 @@
+.barchart {
+  width: 90%;
+  height: 12em;
+  margin: 0 auto;
+  display: flex;
+  padding: $gap;
+  box-sizing: border-box;
+  align-items: flex-end;
+  margin-bottom: 1em;
+}
+
+.bcBar {
```

```suggestion
.bc-bar {
```

This prompted me to update the style guide for OnDemand.